travis/build-samples.sh

```diff
@@ -18,34 +18,50 @@ function usage() {
 
 function buildProject() {
     dir=$1
+    JHIPSTER=$dir
```

Comment:
This script is not used by Travis at all. It was done by one of our contributors to test locally, before submitting a PR

Very very quick look right now, it seems OK. I will review in details later ;)